pkg/systemd/services.html

```diff
@@ -19,27 +19,27 @@
         {{#table_head}}
           <thead>
             <tr>
-              <th scope="col">{{Id}}</th>
+              <th scope="col" class="service-unit-id">{{Id}}</th>
```

Comment:
These aren't being used anywhere, in CSS nor tests. Is that planned?

Reply:
@garrett ?

Reply:
I had them in CSS at some point.

This specific one was being used for the width calculation. The CSS was targeting both the `th` and `td` for fixing the width... I looked and found in in the CSS locally, but updated the branch and don't see it anymore; I guess that changed at the end of last week?

Reply:
Also, I come across unclassed `<th>`s,` <td>`s all the time in Cockpit and see people target them with `:first-child`, `:last-child`, `:nth-child()` and such — we shouldn't target tables (or most elements) like this, unless it's really supposed to be the first or last element and not a specific row. That's why I added it.

(Although, we can fix it on the fly if we do need to target a specific element too.)